public/main.js

```diff
@@ -0,0 +1,138 @@
+//ИМПОРТ КЛАССОВ, ПОЧЕМУ ТО, СДЕЛАТЬ НЕ ВЫШЛО. бЫЛО ПЕРЕПРОБЫВАНО ОЧЕНЬ МНОГО. ТАК ЧТО, ИЗВИНИТЕ, НО ОНИ ЗДЕСЬ
+const search_btn = document.getElementById('search_btn');
+const parentBlock = document.getElementById('music_list');
+const libraryStorage = document.getElementById('libraryStorage');
+//ЗАПРОС НА СЕРВЕР
+
+class HttpRequests {
+    //полная ссылка с методом
+    _requestURL = '';
+    //мой апи для lastfm
+    _apiKey = '';
+    //API root URL
+    _api = '';
+
+    constructor(apiKey, api) {
+        this.apiKey = apiKey;
+        this.api = api;
+    }
+    //Делает запрос на сервер
+    async sendRequests(url, data) {
+        try{
+            this.response = await fetch(url, {
+                method: data.method,
+                body: data.body
+            });
+            return await this.response.json();
+        }
+        catch(error){
+            console.error(error)
+        }
+    }
+    /*
+        Осуществляет поиск трека
+        {string} track - название трека
+    */
+    async searchTrack(track) {
+        this.requestURL = `${this.api}/2.0/?method=track.search&track=${track}&api_key=${this.apiKey}&limit=20&format=json`;
+        return await this.sendRequests(this.requestURL, { method: "GET" });
+    }
+}
+// ОТРИСОВКА ДАННЫХ С СЕРВЕРА
+class Track {
+    track ={
+        image: '',
+        treckname: '',
+        author: ''
+    }
+
+    constructor(track) {
+        this.track = track;
+    }
+    // шаблон отрисовки
```

Comment:
Не корректный комментарий для автодоки
должно было быть как-то так
```suggestion
    /**
     * шаблон отрисовки
     */
```